spec/statsd_spec.rb

```diff
@@ -1,919 +1,856 @@
-# frozen_string_literal: true
-require_relative 'helper'
-require 'allocation_stats' if RUBY_VERSION >= "2.3.0"
-require 'socket'
-require 'stringio'
-require 'time'
-
-SingleCov.covered! file: 'lib/datadog/statsd.rb' if RUBY_VERSION > "2.0"
+require 'spec_helper'
 
 describe Datadog::Statsd do
-  class Datadog::Statsd
-    # we need to stub this
-    attr_accessor :socket
-    # we need to be able to access this
-    attr_accessor :telemetry
+  let(:socket) { FakeUDPSocket.new }
+
+  subject do
+    described_class.new('localhost', 1234,
+      namespace: namespace,
+      sample_rate: sample_rate,
+      tags: tags,
+      logger: logger,
+      telemetry_flush_interval: -1,
+    )
   end
 
-  let(:namespace) { nil }
+  let(:namespace) { 'sample_ns' }
   let(:sample_rate) { nil }
-  let(:socket) { FakeUDPSocket.new }
+  let(:tags) { %w[abc def] }
```

Comment:
Because it doesn't harm, let's use tags just slightly more realistic: for instance `abc:def host:myhost12-11`, WDYT?

Reply:
Actually you did it right after, nvm.